src/components/TalkDetails.tsx

```diff
@@ -28,9 +36,31 @@ export const TalkDetails: React.FC<{ id: string }> = ({ id }) => {
   const talk = data.talk;
   if (!talk) return <p>"Not Found"</p>;
 
+  console.log(data);
+
   return (
     <div>
-      <pre>{JSON.stringify(data, null, 1)}</pre>
+      <h2
+        className={css`
+          font-weight: bold;
+          font-size: 2rem;
+          padding-bottom: 10px;
+        `}
+      >
+        {talk.title}
+      </h2>
+      <AgendaTalksListItemSpeakers speakers={talk.speakers} />
+      <VSpace />
+      <p
+        className={css`
+          font-weight: bold;
```

Comment:
nit: I don't think bold is the right way to display this (especially because they're sometimes on the longer side). We can revisit styling later though!

Reply:
Yeah, the styling is definitely not perfect.  I figured this would be re-addressed when the markdown rendering is in place since this text is mostly markdown.